static/include/micro-rdk.md

```diff
@@ -8,12 +8,14 @@ The micro-RDK is a lightweight version of the {{% glossary_tooltip term_id="rdk"
 
 The only microcontroller the micro-RDK currently supports is the [ESP32](https://www.espressif.com/en/products/socs/esp32).
 
-[Client API](/program/apis/) usage with the Micro-RDK is currently limited to the following supported {{< glossary_tooltip term_id="resource" text="resources" >}}:
+[Client API](/program/apis/) usage with the micro-RDK currently supports only the following {{< glossary_tooltip term_id="resource" text="resources" >}}:
```

Comment:
Was the supported hardware  (ESP32-WROOM/ESP32-WROVER) moved from this page? I'd like them everywhere that makes sense - (install page, the esp32 board model page, and the landing pages) so that users don't buy things that won' work with our beta stack. Sorry for the duplication! Hopefully this will be gone be GA

Reply:
This is just one of the includes and it is used in conjunction with the hardware includes on the landing pages! This page is not apprearing solo anywhere, it's just a reusable snippet. So I think ok? But I'll check rn to make sure hardware is showing up in enough places